ui-v2/app/styles/components/healthcheck-output.scss

```diff
@@ -2,3 +2,10 @@
 .healthcheck-output {
   @extend %healthcheck-output;
 }
+%healthcheck-output em::before {
+  width: 250px;
+  /* TODO: All tooltips previously used */
```

Comment:
Do you typically leave TODOs in here?

Reply:
Yup, nothing is ever considered finished 😄 Plus a lot of times with module systems like this is you find a usecase that doesn't quite work right, in these cases you want to avoid going straight in and modifying the system as you could affect other things without noticing (we have no CSS regression testing as yet). Far safer to tweak here and add a TODO for later.

Generally, they are just little notes for me or someone else to come back to at a later date. They get compiled out of the production code.